bower.json

```diff
@@ -14,22 +14,49 @@
   "homepage": "https://github.com/PolymerElements/app-route",
   "private": true,
   "ignore": [],
+  "dependencies": {
+    "iron-location": "PolymerElements/iron-location#2.0-preview",
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
+  },
   "devDependencies": {
-    "paper-styles": "polymerelements/paper-styles#^1.0.13",
-    "paper-input": "polymerelements/paper-input#^1.1.2",
-    "web-component-tester": "^4.0.0",
-    "iron-component-page": "polymerelements/iron-component-page#^1.0.0",
-    "iron-test-helpers": "polymerelements/iron-test-helpers#^1.0.0",
-    "iron-pages": "PolymerElements/iron-pages#^1.0.7",
-    "paper-card": "PolymerElements/paper-card#^1.1.1",
-    "paper-icon-button": "polymerelements/paper-icon-button#^v1.0.0",
-    "paper-toggle-button": "polymerelements/paper-toggle-button#^v1.0.0",
-    "google-youtube": "GoogleWebComponents/google-youtube#^1.2.1",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.2.0",
-    "paper-spinner": "PolymerElements/paper-spinner#^1.1.1"
+    "google-youtube": "GoogleWebComponents/google-youtube#2.0-preview",
+    "iron-component-page": "polymerelements/iron-component-page#2.0-preview",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#2.0-preview",
+    "iron-pages": "PolymerElements/iron-pages#2.0-preview",
+    "iron-test-helpers": "polymerelements/iron-test-helpers#2.0-preview",
+    "paper-card": "PolymerElements/paper-card#2.0-preview",
+    "paper-icon-button": "polymerelements/paper-icon-button#2.0-preview",
+    "paper-input": "polymerelements/paper-input#2.0-preview",
+    "paper-spinner": "PolymerElements/paper-spinner#2.0-preview",
+    "paper-styles": "PolymerElements/paper-styles#2.0-preview",
+    "paper-toggle-button": "polymerelements/paper-toggle-button#2.0-preview",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
```

Comment:
you can remove this dependency from both variants since `wct` already has it & injects it for you 👌